xml/bp_troubleshooting_cephfs.xml

```diff
@@ -271,4 +271,450 @@ sudo zypper in ceph-common
+           <entry>0x200000000000000</entry>
+         </row>
+         <row>
+           <entry>CRUSH_TUNABLES5</entry>
+           <entry>58</entry>
+           <entry>0x400000000000000</entry>
+         </row>
+         <row>
+           <entry>NEW_OSDOPREPLY_ENCODING</entry>
+           <entry>58</entry>
+           <entry>0x400000000000000</entry>
+         </row>
+         <row>
+           <entry>FS_FILE_LAYOUT_V2</entry>
+           <entry>58</entry>
+           <entry>0x400000000000000</entry>
+         </row>
+         <row>
+           <entry>FS_BTIME</entry>
+           <entry>59</entry>
+           <entry>0x800000000000000</entry>
+         </row>
+         <row>
+           <entry>FS_CHANGE_ATTR</entry>
+           <entry>59</entry>
+           <entry>0x800000000000000</entry>
+         </row>
+         <row>
+           <entry>MSG_ADDR2</entry>
+           <entry>59</entry>
+           <entry>0x800000000000000</entry>
+         </row>
+         <row>
+           <entry>OSD_RECOVERY_DELETES</entry>
+           <entry>60</entry>
+           <entry>0x1000000000000000</entry>
+         </row>
+         <row>
+           <entry>CEPHX_V2</entry>
+           <entry>61</entry>
+           <entry>0x2000000000000000</entry>
+         </row>
+         <row>
+           <entry>RESERVED</entry>
+           <entry>62</entry>
+           <entry>0x4000000000000000</entry>
+         </row>
+       </tbody>
+     </tgroup>
+   </table>
+   <para>
+     A possible solution to allow an old kernel client to mount a recent
+     &cephfs; is to modify the cluster CRUSH profile. CRUSH profiles define a
+     set of CRUSH tunables that are named after the &ceph; versions in which they
+     were introduced. For example, the 'firefly' tunables are first supported in
+     the Firefly release (0.80) and older clients will not be able to access the
+     cluster. Thus, to fix the problem shown above, the following command can be
+     used:
+   </para>
+<screen>
+&prompt.cephuser;ceph osd crush tunables hammer
+</screen>
+   <para>
+     This will adjust the CRUSH profile to the behaviour it had for the Hammer
+     (0.94) release. Note however that this is not the optimal behaviour for the
+     cluster. To change back to the optimal profile, run the following command:
+   </para>
+<screen>
+&prompt.cephuser;ceph osd crush tunables optimal
+</screen>
+   <para>
+     The following table lists the available CRUSH profiles and which CRUSH
+     tunables versions (the CRUSH_TUNABLE feature bits in the previous table)
+     they correspond to. It also identifies the minimum kernel version required
+     to use for each profile. Note however that Operating System vendors may
+     choose to backport features to their kernels, so these kernel versions are
+     valid for mainline kernels only. The kernel client included in &sls; 15
+     SP2, for example, includes backports of features and bug fixes relevant for
+     usage in SES7 clusters.
```

Comment:
@asettle -- what do you think, entities here too or leave the short form?

Reply:
Hmm, let's leave short form for now :)